clusterloader2/testing/density/rc.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: {{.Basename}}
+spec:
+  replicas: {{.Replicas}}
+  selector:
+    name: {{.Basename}}
+  template:
+    metadata:
+      labels:
+        name: {{.Basename}}
+        app: pause
+    spec:
+      containers:
+      - image: gcr.io/kubernetes-e2e-test-images/pause:3.1
```

Comment:
I think we're just using `k8s.gcr.io/pause:3.1` in our tests. Not sure what's the difference though.